unity-ktwo/Assets/Scripts/Behaviours/DishWashBehaviour.cs

```diff
@@ -13,11 +14,13 @@ public class DishWashBehaviour : MonoBehaviour
     private Vector3 offsetFromPlayer;
     private bool active = true;
     private HashSet<GameObject> affectedEntities = new HashSet<GameObject>();
+    private ParticleSystem bubbles;
 
     void Start()
     {
-        player = PlayerManager.instance.player;
+        player = (!isServer) ? PlayerManager.instance.players[0] : PlayerManager.instance.player;
```

Comment:
Hm, we probably need to take another look at this. Originally this was used to debug the bubble ability when we couldn't get it working at all. Left hard-coded like this will probably mean that the dish wash ability won't work if anyone else other than player 1 plays as chef and uses the dish wash ability.

Reply:
We would need to come up with a way for the server to figure out which player to parent it to essentially.